### ISSUE TITLE
sbt import hangs in 1.3.0 with AdoptOpenJDK 8.0.232.09

### ISSUE THREAD

**Issue by mchmaj** [sbt/sbt#5253]
## steps
sbt version: 1.3.0
AdoptOpenJDK 8.0.232.09

## problem
Sbt import hangs on an empty project.
`[info] Loading settings for project global-plugins from idea.sbt ...` appears and then nothing.
The same problem appears with sbt 1.3.4.
Works fine with sbt 1.2.8.
Also works fine with sbt 1.3.0 and Oracle jdk 1.8.0_45 or with AdoptOpenJDK 11.

**Comment by eed3si9n** [comment]
@mchmaj Thanks for the report.

For what it's worth we use AdoptOpenJDK 8.0.222 in a part of build matrix. https://travis-ci.org/sbt/sbt/jobs/619111059#L209-L212

```
0.07s$ export JAVA_HOME="$JABBA_HOME/jdk/$TRAVIS_JDK" && export PATH="$JAVA_HOME/bin:$PATH" && java -Xmx32m -version
openjdk version "1.8.0_222"
OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_222-b10)
OpenJDK 64-Bit Server VM (AdoptOpenJDK)(build 25.222-b10, mixed mode)
```

**Comment by mchmaj** [comment]
Yup, switching to AdoptOpenJDK 8.0.222 helps. Still, it would be good to have it working for the newest version as well.

**Comment by eed3si9n** [comment]
I am playing around with my Docker container using 1.8.0_232 and things look ok.

**Comment by eed3si9n** [comment]
I just updated on my mac to 1.8.0_232 to test this and I still can't reproduce this.

```
$ java -version
openjdk version "1.8.0_232"
OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_232-b09)
OpenJDK 64-Bit Server VM (AdoptOpenJDK)(build 25.232-b09, mixed mode)
$ sbt
[info] Loading settings for project global-plugins from idea.sbt,vimquit.sbt,plugins.sbt,metals.sbt,pgp.sbt ...
[info] Loading global plugins from /Users/eed3si9n/.sbt/1.0/plugins
[info] Loading project definition from /private/tmp/hello/project
[info] Loading settings for project root from build.sbt ...
[info] Set current project to hello (in build file:/private/tmp/hello/)
[info] sbt server started at local:///Users/eed3si9n/.sbt/1.0/server/478e6db75688771ddcf1/sock
sbt:hello> exit
[info] shutting down sbt server
```

What operating system are you using? This might be relevant since some JDK bugs are macOS version dependent (https://github.com/AdoptOpenJDK/openjdk-build/issues/1206). Another thing to try might be to specify `-Dsbt.global.base=/tmp/sbt/` to make sure it's not caused by one of the global plugins.

**Comment by mchmaj** [comment]
Sorry for the delay. I'm using Win 7, so the macOS-related bugs are not at fault here.

**Comment by eed3si9n** [comment]
ok. Then you could be hitting some issue on directories-jvm used by Coursier.
See for example https://github.com/soc/directories-jvm/issues/28.

**Comment by eed3si9n** [comment]
Could you try using the latest sbt 1.3.7?

**Comment by mchmaj** [comment]
1.3.7 is actually even worse on my machine, it hangs for Oracle jdk 1.8.0_45 and AdoptOpenJDK 11.0.4 as well.

**Comment by eed3si9n** [comment]
I am not sure but if it's getting stuck calling out to powershell, setting `-Dsbt.coursier.home` property or setting `COURSIER_HOME` environment variable might help.

**Comment by mchmaj** [comment]
The hanging also happens for 1.3.8 with AdoptOpenJDK 11.0.4.11.

Also, the project has courser disabled, so I'm not sure if any coursier-related properties are relevant here.

**Comment by eed3si9n** [comment]
@mchmaj How are you disabling Coursier? Even if the proper build is using Ivy, the metabuild (`project/pluigins.sbt`) could be using Coursier.

There's a report saying that antivirus scan kicks in https://github.com/soc/directories-jvm/issues/28, which I think can be avoided if `COURSIER_HOME` is set.

**Comment by mchmaj** [comment]
It seems that setting the `COURSIER_HOME` environmental variable changes nothing for me. Setting `-Dsbt.coursier.home`, on the other hand, causes sbt to go one step further than before - instead of getting stuck on loading settings for global plugins, it starts actually loading plugins and gets stuck there.

**Comment by eed3si9n** [comment]
Closing this for now. Please open a new issue if this still happens with the latest stable 1.x.
